### PR TITLE
Embed options: input code does not fit

### LINKED ISSUE #2398: embed options: code does not fit
ff 
embed

### PR DESCRIPTION
**Pull request by kandre**
Fix #2398 

[Test link](https://mf-geoadmin3.dev.bgdi.ch/kan_embed_input_css/)

### CODE REVIEW

**Comment by kandre** [comment]
@oterral I put a bit more of 170/155px to insure ^Benutzerdefiniert Grösse' label better fits

File: src/components/share/style/share.less [hunk 1]
```diff
@@ -105,10 +101,17 @@
   }
 
   .ga-user-size {
+    .ga-embed-input-modal {
+      width: 100%!important;
```
Comment (oterral) [inline src/components/share/style/share.less]: the !important is not needed

**Comment by kandre** [comment]
jenkins test

**Comment by kandre** [comment]
jenkins test

**Comment by oterral** [comment]
Tested on ff, ie11 and chrome , works nicely

**Comment by oterral** [comment]
jenkins test

**Comment by kandre** [comment]
Thks

**Pull request merged**